client/me/security-account-recovery/edit-phone.jsx

```diff
@@ -55,8 +56,8 @@ class SecurityAccountRecoveryRecoveryPhoneEdit extends Component {
 					isDeletable={ havePhone }
 					saveText={ this.props.translate( 'Save Number' ) }
 					onSave={ this.onSave }
-					onDelete={ this.onDelete }
-					onCancel={ this.onCancel }
+					onDelete={ this.props.onDelete }
+					onCancel={ this.props.onCancel }
```

Comment:
Not related to this PR, but I noticed the logic for `isSavable` seems a bit off. The "Save" button becomes active even when only the country code is changed and the phone number field is still empty. It looks like we should also check whether `this.state.phoneNumber.phoneNumber` is null or not to ensure a valid phone number has been entered.

Reply:
Updated the conditional to

```
	if ( ! this.state.phoneNumber.phoneNumber || ! this.state.phoneNumber.phoneNumberFull ) {
			return false;
	}
```